### ISSUE TITLE
Code from example page?

### ISSUE THREAD

**Issue by dantaeyoung** [littleredcomputer/odex-js#2]
This is wonderful! Could you include the code from your example page (http://blog.littleredcomputer.net/math/odex/js/2016/04/03/lotka-volterra.html)? I'm particularly interested in how you're plotting the phase space.

**Comment by littleredcomputer** [comment]
Thanks for writing!

The source for all the demos is in my github-pages repo.  So from here
<https://github.com/littleredcomputer/littleredcomputer.github.io/blob/master/_posts/2016-04-03-lotka-volterra.md>
we
can see that the graph is drawn by odexdemo.PredatorPrey, and the source
for that is here
<https://github.com/littleredcomputer/littleredcomputer.github.io/blob/d1ab07b51d2589f9fd282aabba6355410f77f747/src/odex-demo.ts#L101>,
which delegates to phaseDraw
<https://github.com/littleredcomputer/littleredcomputer.github.io/blob/d1ab07b51d2589f9fd282aabba6355410f77f747/src/odex-demo.ts#L23>
.

(I used this as an opportunity to dabble with Typescript. I enjoyed it but
it is probably overkill just for the demo.)

So for the population, we draw the points (t, x(t)), (t, y(t)); for the
phase, we draw the one point (x(t), y(t)).

kind regards

**Comment by littleredcomputer** [comment]
Thanks for writing!

The source for all the demos is in my github-pages repo.  So from [here](https://github.com/littleredcomputer/littleredcomputer.github.io/blob/master/_posts/2016-04-03-lotka-volterra.md) we can see that the graph is drawn by odexdemo.PredatorPrey, and the sourcefor that is [here](https://github.com/littleredcomputer/littleredcomputer.github.io/blob/d1ab07b51d2589f9fd282aabba6355410f77f747/src/odex-demo.ts#L101), which delegates to [phaseDraw](https://github.com/littleredcomputer/littleredcomputer.github.io/blob/d1ab07b51d2589f9fd282aabba6355410f77f747/src/odex-demo.ts#L23).

(I used this as an opportunity to dabble with Typescript. I enjoyed it but it is probably overkill just for the demo.)

So for the population, we draw the points (t, x(t)), (t, y(t)); for the phase, we draw the one point (x(t), y(t)).
